spec/spec_helper.rb

```diff
@@ -126,6 +126,10 @@
   config.default_max_wait_time = 4
 end
 
+Capybara::Screenshot.register_filename_prefix_formatter(:rspec) do |example|
+  "screenshot_#{example.description.downcase.tr(' ', '-').gsub(/^.*\/spec\//, '')}"
```

Comment:
Are we at risk of other problematic characters here too? quotes strike me as a likely candidate. I wonder if it's safer to change all non ascii values?